aspnetcore/blazor/debug.md

```diff
@@ -36,11 +36,11 @@ Development of further debugging scenarios is an on-going focus of the engineeri
 
 To debug a Blazor client-side app in Chrome:
 
-* Build a Blazor app in `Debug` configuration (the default for unpublished apps).
-* Run the Blazor app in Chrome (version 70 or later).
-* With the keyboard focus on the app (not in the developer tools panel, which you should probably close for a less confusing debugging experience), select the following Blazor-specific keyboard shortcut:
-  * `Shift+Alt+D` on Windows/Linux
-  * `Shift+Cmd+D` on macOS
+1. Run a Blazor client-side app in `Debug` configuration. Pass the `--configuration Debug` option to the [dotnet run](/dotnet/core/tools/dotnet-run) command: `dotnet run --configuration Debug`.
+1. Access the app in Chrome (version 70 or later) or with a Microsoft Edge preview ([Edge Dev Channel](https://www.microsoftedgeinsider.com)).
+1. With the keyboard focus on the app (not in the developer tools panel, which you should probably close for a less confusing debugging experience), select the following Blazor-specific keyboard shortcut:
```

Comment:
Way to long for MT. Break up in two sentences.  You should probably have this instruction earlier. Something like:
Debug with keyboard focus on the app, not the dev tools panel. The dev tools panel should typically be closed when debugging ...

Reply:
Or 
Client side debugging requires chrome 70 or later, or Edge Dev ...